docs/getting-started/understanding-config-files.md

```diff
@@ -107,12 +138,58 @@ Please be aware that this procedure is specific to the topology configuration fi
 the node configuration file. Additionally, the SIGHUP signal will prompt the system to
 re-read the block forging credentials file paths and attempt to fetch them to initiate
 block forging. If this process fails, block forging will be disabled. To re-enable block
-forging, ensure that the necessary files are present.
+forging, ensure that the necessary files are present (check [here](../../reference/dynamic-block-forging)
+for more details about dynamically enabling/disabling block forging).
 
 You can disable ledger peers by setting the `useLedgerAfterSlot` to a negative
 value.
 
-If you are syncing the network for the first time, please connect to IOG relays at `relays-new.cardano-mainnet.iohkdev.io` by adding them to your local root peers. Additionally, set `useLedgerAfterSlot` to `0` to disable ledger peers. You have the flexibility to use different relays, but it's important to ensure you trust them to provide an honest chain. Once your node is fully synced, feel free to remove these relays from your local root peers. Also, make it a practice to manually verify if other stake pool relays are on the same chain as yours. When you enable ledger peers by setting `useLedgerAfterSlot`, your node will establish connections with relays registered on the chain. It will actively churn through these peers by randomly selecting new ones based on stake distribution while also removing the least performing 20%.
+If you are syncing the network for the first time, please connect to IOG
+relays at `relays-new.cardano-mainnet.iohkdev.io` by adding them to your local
+root peers. Additionally, set `useLedgerAfterSlot` to `-1` to disable ledger
+peers. You have the flexibility to use different relays, but it's important to
+ensure you trust them to provide an honest chain. Once your node is fully
+synced, feel free to remove these relays from your local root peers. Also,
+make it a practice to manually verify if other stake pool relays are on the
+same chain as yours. When you enable ledger peers by setting
+`useLedgerAfterSlot`, your node will establish connections with relays
+registered on the chain. It will actively churn through these peers by
+randomly selecting new ones based on stake distribution while also removing
+the least performing 20%.
+
+### Bootstrap Peers
```

Comment:
Check with @kevinhammond what is the official name for bootstrap peers and use `# Official (Bootstrap peers)` as the title.